src/current/v23.2/hashicorp-integration.md

```diff
@@ -62,8 +63,27 @@ CockroachDB users can use Vault's PostgreSQL Database Secrets Engine to handle t
 
 Try the tutorial: [Using HashiCorp Vault's Dynamic Secrets for Enhanced Database Credential Security in CockroachDB]({% link {{ page.version.version }}/vault-db-secrets-tutorial.md %})
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/sql/grant-revoke-schema-changes.md %}
```

Comment:
Do we need  more of a highlight of why it's important? I worry people will just gloss over this without reading that section.

Reply:
I agree that's a valid concern.  To call this out more, I made it into its own subsection called 'How to speed up user/role management' so it shows up in the table of contents and looks more important (I hope) while still being in the existing "Use Vault's PostgreSQL Database Secrets Engine to manage CockroachDB SQL users and their credentials" section

Please let me know what you think!